libmscore/note.h

```diff
@@ -119,6 +119,7 @@ class NoteHead final : public Symbol {
             HEAD_H,
             HEAD_H_SHARP,
 
+            HEAD_CUSTOM,
```

Comment:
Watch out, the addition of this custom type can have weird implication since HEAD_GROUPS is used to "loop through" the enum. I believe I catch all problems but still, review welcome.

Reply:
Sure, I'm working on it. What is the best way to introduce changes? Should I make a PR based on your changes?

Reply:
Sure, you can do a PR based on my branch. Let's talk when you are done.